src/data/full-quiz.js

```diff
@@ -1963,7 +1963,7 @@ const fullQuiz = [
   },
   {
     "Question": "How can you switch to a new branch in Git?",

```

Comment:
I propose making a change to the question too.
How can you create a new branch and switch to it in Git?

Reply:
yeah that looks better ;)